pages/common/mpg321.md

```diff
@@ -0,0 +1,33 @@
+# mpg321
+
+> High Performance MPEG 1.0/2.0/2.5 Audio Player for Layer 1, 2, and 3.
+> Mpg321 was written (sometime in 1999) to be a drop-in replacement for the (previously) non-free mpg123 player.
+> More information: <http://mpg321.sourceforge.net/>.
+
+- Play an audio source exactly N=3 times:
+
+`mpg321 -l {{3}} {{path/to/file_a|URL}} {{path/to/file_b|URL}} {{...}}`
+
+- Play an audio source forever (0 means until interrupted):
+
+`mpg321 -l 0 {{path/to/file_a|URL}} {{path/to/file_b|URL}} {{...}}`
```

Comment:
I think we can combine these 2 examples into one. We can just mention `-l {{3}}` and write the description as - "Play an audio source 3 times (0 means forever):"

Reply:
ok. Replaces the 2 lines with: 'Play an audio source exactly N times (N=0 means forever)'